services/worker-manager/src/api.js

```diff
@@ -420,9 +437,9 @@ builder.declare({
   });
 });
 
-builder.declare({
+declareWithTrailingColon({
   method: 'get',
-  route: '/workers/:workerPoolId:/:workerGroup/:workerId',
+  route: '/workers/:workerPoolId/:workerGroup/:workerId',
   name: 'worker',
   scopes: 'worker-manager:get-worker:<workerPoolId>/<workerGroup>/<workerId>',
   title: 'Get a Worker',
```

Comment:
I'm not sure if it is documented anywhere else, but maybe we should add also in description that `workerPoolId` should be escaped/urlencoded before by the clients? Or is it too obvious? :)

Reply:
I think we've documented that all parameters need to be urlencoded, and the various clients take care of this.

Reply:
great then!

Reply:
Well, I say that, but now I can't find it here: https://community-tc.services.mozilla.com/docs/manual/using/api

Reply:
That's definitely where it should be documented.  You're right that in the bug @jwmoss identified (https://bugzilla.mozilla.org/show_bug.cgi?id=1560645) we added the flexibility to allow an unencoded _final_ component, but that's not allowed on non-final components.